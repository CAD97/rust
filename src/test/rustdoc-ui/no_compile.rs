// check-pass

// compile-flags:--test --test-args=--include-ignored --test-args=--test-threads=1
// normalize-stdout-test: "src/test/rustdoc-ui" -> "$$DIR"
// normalize-stdout-test "finished in \d+\.\d+s" -> "finished in $$TIME"

/// These code blocks should be treated as rust code in documentation, but never
/// treated as a test and compiled, even when testing ignored tests.
///
/// ```rust,no_compile
/// // haha rustc
/// i"have" 0utsmarted k#thee
/// ```
///
/// ```no_compile
/// // haha rustc
/// i"have" 0utsmarted k#thee
/// ```
///
/// For comparison, ignore does get run:
///
/// ```ignore (but-actually-we-include-ignored)
/// println!("Hello, world");
/// ```
struct S;
