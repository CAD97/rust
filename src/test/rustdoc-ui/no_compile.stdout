
running 1 test
test $DIR/no_compile.rs - S (line 22) ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in $TIME

